drivers/leviton/acquisuite_spec.cr

```diff
@@ -2,23 +2,24 @@ require "placeos-driver/spec"
 
 DriverSpecs.mock_driver "Leviton::Acquisuite" do
   headers = {"Content-Type" => ["multipart/form-data; boundary=MIME_BOUNDRY_MIME_BOUNDRY_MIME_BOUNDRY"]}
-  test_devices = ["mb-001", "mb-002"]
+  test_devices = ["mb-001"]
 
   # First, we need to receive some failed LOGFILEUPLOAD requests to work out our list of devices
   test_devices.each do |device_name|
-    dev_log = File.read("/app/repositories/local/drivers/leviton/#{device_name}.log")
+    dev_log = File.read("/app/repositories/local/drivers/leviton/#{device_name}.63BD5AFD_2.log.gz")
```

Comment:
Is this supposed to be specific 63BD5AFD_2?

Reply:
Yeah